components/brave_news/browser/test/BUILD.gn

```diff
@@ -0,0 +1,43 @@
+# Copyright (c) 2021 The Brave Authors. All rights reserved.
```

Comment:
2023

Reply:
I don't think this should actually be changed, as the file has been moved